src/Components/Common/Header.css

```diff
@@ -0,0 +1,13 @@
+header {
+    padding: 20px;
+    justify-content:center; /* Distributes space between elements */
+    align-items: center; /* Aligns elements vertically */
+  }
+  
+  h1 {
```

Comment:
h1 color is a bit dark and hard to read on the black background. maybe lighten :)